src/Select.js

```diff
@@ -54,6 +89,45 @@ export default class Select extends React.Component {
     if (this.props.options.length > 5 && this.refs.selected && prevState.active === false && this.state.active === true) {
       this.refs.selected.scrollIntoView();
     }
+
+    // when showing drop down, update the positioning styles
+    if (this.props.scrollParentClass && !prevState.active && this.state.active) {
+      this.setState(this.getDropdownStyle());
+    }
+  }
+
+  getDropdownStyle() {
+    let parent = ReactDOM.findDOMNode(this.refs.parent);
+    let source = parent;
+    let offsetTop = 0;
+    let scrollParentTop = this.scrollParent.scrollTop;
+    while (source) {
+      offsetTop += source.offsetTop;
+      source = source.offsetParent;
+    }
+    return {
+      fixedDropdownStyle: {
+        width: `${parent.offsetWidth}px`,
+        position: 'fixed',
+        top: `${(offsetTop - scrollParentTop) + parent.offsetHeight}px`
+      }
+    };
+  }
+
+  handleScrollParent() {
+    let state = Object.assign({}, this.getDropdownStyle(), {active: false});
```

Comment:
No we don't, I think we could put that update inside toggleActive() when active is true